misc/scratch/remote-dev.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "vs code remote dev",
+  "launch_script": "true",
+  "instance_type": "c6a.4xlarge",
+  "ami": "ami-09d56f8956ab235b3",
+  "size_gb": 100,
```

Comment:
I also quickly ran out of disk with only 50g, so doubling for regular remote dev.